packages/core/template/default/index.md

```diff
@@ -1,132 +1,75 @@
+You have just initialized a _default_ MarkBind site, which is equipped with a set of core features of a MarkBind site like Site and Page Navigation, as well as some links to our User Guide, to help you get started more conveniently!
 
-**A list:**
+<box type="tip">
 
-* item A
-* item B
-  1. item b1
-  1. item b2
-  1. item b3
+If you were intending to convert an existing GitHub wiki or a docs folder into MarkBind, use the "--convert" flag instead. See <a href="https://markbind.org/userGuide/markBindInTheProjectWorkflow.html#converting-existing-project-documentation-wiki" target="_blank">User Guide: MarkBind in the Project Workflow</a> for more information.
 
-**A `code` example:**
+If you want to start with a <tooltip content="i.e. without any content">_minimal_</tooltip> template instead, use the "--template" flag to initialize a minimal site instead of the default. See <a href="https://markbind.org/userGuide/templates.html" target="_blank">User Guide: Templates</a> for more information.
 
-```html
-<foo>
-  <bar type="name">goo</bar>
-</foo>
-```
+</box>
 
-## Sub Heading 1.1
+---
 
-A <tooltip content=":exclamation: some **important explanation**" placement="top" trigger="hover">tooltip</tooltip>, a <trigger for="modal:modalinfo" trigger="click">modal</trigger>, a <a href="https://markbind.org/">link</a>, a <span class="badge bg-danger">badge</span>, another <span class="badge bg-warning text-dark">badge</span>.
+## Navigating this site
 
-<modal header="Modal Title" id="modal:modalinfo">
-Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
-</modal>
+This _default_ site comes pre-configured with the core <a href="https://markbind.org/userGuide/components/navigation.html#navigation-components" target="_blank">Navigation components</a>: a <tooltip content="Site Navigation">**siteNav**</tooltip>, a <tooltip content="Page Navigation">**pageNav**</tooltip>, a <tooltip content="Navigation Bar">**NavBar**</tooltip>, and a **Search Bar**</tooltip>. To help you get started with the **siteNav**, we have included <tooltip content="Topic 1, Topic 2, Topic 3, Topic 3a, Topic 3b">five dummy placeholder pages</tooltip>. The **NavBar** also comes with a placeholder slot for your custom Logo.
```

Comment:
```suggestion
This _default_ site comes pre-configured with the core <a href="https://markbind.org/userGuide/components/navigation.html#navigation-components" target="_blank">Navigation components</a>: a <tooltip content="Site Navigation">**siteNav**</tooltip>, a <tooltip content="Page Navigation">**pageNav**</tooltip>, a <tooltip content="Navigation Bar">**NavBar**</tooltip>, and a **Search Bar**. To help you get started with the **siteNav**, we have included <tooltip content="Topic 1, Topic 2, Topic 3, Topic 3a, Topic 3b">five dummy placeholder pages</tooltip>. The **NavBar** also comes with a placeholder slot for your custom Logo.
```

Reply:
Thanks for the find! I didn't notice that :sweat_smile: 

Updated in [fab85db](https://github.com/MarkBind/markbind/pull/2225/commits/fab85db23ed5ba23eacde0a78dc99c45d63d69c7).